### ISSUE TITLE
RA Comments- Manual #3

### ISSUE THREAD

**Issue by mailemccann** [AguaClara/ram_pump#70]
Sorry this is SO late. Here are my comments for your third manual. Overall, I think this was really great!! I don't see much to change.

- Lit Review- I agree with @sidneylok , refer to your figure in the paragraph before or after its shown
- You can specify that the force closing the plate is **drag force** from the flowing water
- Fab Details- I think you could make the parts list into a table, and have the link to purchase in a column, or put the link under the initial description for the part as a bullet point
- #5 in the first procedure has a different sentence structure than the other steps (just the first sentence) Maybe restructure this for readability? This is nitpicking, I know.
- In Measuring the Flow Rate, I would explicitly state the limitations of this method, how human error could be involved, and how you should run it many times to ensure that you get repeatable results
- In conclusion, quantify "very inefficient"- in the form of a fraction, percentage, etc
- Also mention the limitation in measurement instruments- pressure sensors not high enough to measure the actual pressure of the check valve

Awesome job!! Send me an email if you want any advice on adding the info on the Monroe conversation. I think its worth it to add a section for the draft due tomorrow (again, I'm sorry this is so late :/) about hypotheses for whats causing inefficiency!

**Comment by cheertsang** [comment]
Things to fix:
- [x] @everyone address grader and RA comments throughout
- [x] @cheertsang  add to section on the pulley experiments: include new data we got, calculate forces
- [ ] headloss calculations
- [x] @alyssaju include things we learned from meeting with Monroe: new pressure sensors, increasing diameter of effluent check valve
- [ ] fix code in report (the version in the calculations markdown files are the correct versions I think) - verify this
- [x] make parts list into a table
